### PR TITLE
Fix right panel width being reset on ui update

### LINKED ISSUE #339: Opening or reloading jar or mappings resets right panel width
While Enigma keeps the size preferences for the left panel of the UI, this is not the case for the right panel (the one with the wonderful structure tab among others :p). It gets reseted when you opens or reloads a jar or mappings.
However, note that Enigma saves the preferences, as when you close and reopen Enigma, the right panel width is kept before a jar or mapping loading.

### PR DESCRIPTION
**Pull request by YanisBft**
Closes #339

### CODE REVIEW

File: enigma-swing/src/main/java/cuchaz/enigma/gui/Gui.java [hunk 1]
```diff
@@ -923,6 +923,8 @@ public void updateUiState() {
 			splitRight.setRightComponent(logSplit);
 			logSplit.setLeftComponent(tabs);
 		}
+
+		splitRight.setDividerLocation(splitRight.getDividerLocation());
```
Comment (2xsaiko) [inline enigma-swing/src/main/java/cuchaz/enigma/gui/Gui.java:927]: I wonder why that fixes it... that's stupid haha

Comment (YanisBft) [reply enigma-swing/src/main/java/cuchaz/enigma/gui/Gui.java:927]: I think that setting the component also sets the width to the minimum width of the component

**Review by 2xsaiko — APPROVED** [review]
...but it seems to work.

LGTM

File: enigma-swing/src/main/java/cuchaz/enigma/gui/Gui.java (hunk 1 above)
Comment (2xsaiko) [reply enigma-swing/src/main/java/cuchaz/enigma/gui/Gui.java:927]: That would explain it resetting, yeah.

**Pull request merged**